scripts/sign.py

```diff
@@ -142,7 +144,7 @@ def main():
     sig_len = ceil_div(key.size() + 1, 8)
     img_size = len(img)
 
-    hdr_version = 0      # SHDR_VERSION (always 0)
+    hdr_version = args.version # SHDR_VERSION
```

Comment:
`SHDR_VERSION` does not exist anywhere, and should be replaced by `struct shdr_bootstrap_ta::version` IMO.

For maximum clarity, I would even rename `struct shdr_bootstrap_ta::version` to `struct shdr_bootstrap_ta::ta_version` in a separate patch.

Reply:
If no-one argue against, I will append such commit to this P-R.

Reply:
I wonder if not also renaming `sign.py` argument `--version` into `--ta_version`, in case we later need some header version, a script version or whatever?

Reply:
I'm fine with both actually.